test/models/organization_test.rb

```diff
@@ -69,13 +69,13 @@ def organization
   end
 
   test 'has a log' do
-    skip """
+    skip '''
```

Comment:
Combine '' and "\n      Tracks when:\n        - A member invites another member.\n        - An invited member accepts.\n        - A member leaves.\n        - A member is kicked out.\n    " into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.<br>Combine "\n      Tracks when:\n        - A member invites another member.\n        - An invited member accepts.\n        - A member leaves.\n        - A member is kicked out.\n    " and '' into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.